vassal-app/src/main/resources/VASSAL/i18n/VASSAL.properties

```diff
@@ -66,11 +66,21 @@ Dialogs.disable=Don't show this again
 Dialogs.show_details=Show details
 Dialogs.hide_details=Hide details
 Dialogs.dont_show_again=Don't show this again
+
+Dialogs.removed_code_title=Defunct Custom Code
+Dialogs.removed_code_heading=Defunct Custom Code
+Dialogs.removed_code_message=This module uses VASSAL classes, methods, or fields which were deprecated more than a year ago and have since been removed. In order to run this module, uses of the classes, methods, or fields listed below must be replaced and the custom code recompiled.
```

Comment:
This module cannot be opened because it contains "custom code" that references VASSAL classes, methods, or fields which were announced as deprecated more than a year ago and have since been removed. 

IF YOU ARE A MODULE USER - you will need to contact the module designer/maintainer and get them to correct the problems in their custom code. If the module maintainer is no longer active or cannot be found, please post a message in the Technical Support forums at https://vassalengine.org/forum

IF YOU ARE THE MODULE DESIGNER/MAINTAINER - In order to run this module, uses of the classes, methods, or fields listed below must be replaced and the custom code recompiled (alternatively the custom code can be removed from the module). If you are not the original maintainer and do not have the knowledge of how to do this, please post a message in the Technical Support forums at https://vassalengine.org/forum

Reply:
Do you think we should mention for the maintainers that they can find notes about replacements in the Javadoc and provide a link to that?

Reply:
Seems like a reasonably good idea